### DISCUSSION TITLE
Gerenciamentos de credenciais

### DISCUSSION THREAD

**Discussion by victormiguez** [frontendbr/forum#100] [Discussão]
Como vocês lidam com o gerenciamento das credenciais dos projetos?
Por exemplo, login e senha do servidor de produção, staging, etc.
O maior problema que encontro é em saber qual a melhor forma de compartilhar isso com outros integrantes do projeto.
Hoje onde eu trabalho, nós utilizamos o Confluence, geramos um documento para cada projeto e as credenciais ficam lá.
Como vocês lidam com isso? Especialmente se for um projeto open source, qual seria a melhor forma de guardar esses dados?

**Comment by felipefialho** [comment]
Mano, temos usado uma ferramenta chamada 1password.

Funciona bem. Você coloca todas as credenciais e compartilha com o resto da equipe.

**Comment by Raffesmind** [comment]
Aqui usamos o Meldium para fazer essa gerencia de contas entre a equipe, também tem funcionado de forma bem bacana.

**Comment by eduardojmatos** [comment]
Boa @Raffesmind Meldium tem até extensão no Chrome @victormiguez . É bem interessante, mas custa $29 pra 20 usuários. Mesmo assim, ainda acho que vale :)

---

*OBS: só pra constar, vc criou a 100ª issue do forum! Parabéns manolo =D

cc @frontendbr/admins

**Comment by raphaklaus** [comment]
Eu vou à maneira antiga: SSH. Cada membro tem uma chave que é copiada para authorized_keys no servidor.
Quando o cara sair da equipe, é só remover a chave dele de lá. Pronto. Sem usernames e passwords rolando solto por aí. :)

**Comment by eduardojmatos** [comment]
@raphaklaus se for só acesso a servidor, concordo com você. Só add a chave e boas. O Meldium é mais pra gerenciar os acessos a serviços com login e senha entre a equipe

**Comment by jonasporto** [comment]
Trabalhei em uma empresa onde era utilizado o [LastPass](https://lastpass.com/pt/), resolvia muito bem essa questão.

**Comment by thekaduu** [comment]
Aqui na empresa utilizamos o [LastPass](https://lastpass.com/pt/) e funciona muito bem.

**Comment by leonardoelias** [comment]
Aqui trabalhamos usamos 1password, funciona muito bem : )
